src/Game.py

```diff
@@ -32,4 +35,82 @@ def initGame(config1, config2, size):
     else:
         currentPlayer = PLAYER_2
 
-    return boards, currentPlayer
+    return gameState, currentPlayer
+
+def decrementTargetFleetLife(boards, attacker):
+    boardsCopy = deepcopy(boards)
+
+    if attacker == PLAYER_1:
+        boardsCopy[PLAYER_2]["life"] -= 1
+    else:
+        boardsCopy[PLAYER_1]["life"] -= 1
+
+    return boardsCopy
+
+def gameIsWon(boards, player):
+    gameIsWon = True
```

Comment:
No need to use a var. `return false` where you set GameIsWon to `false` and `return true` after the loops